pyanaconda/modules/storage/bootloader/base.py

```diff
@@ -88,7 +88,7 @@ class BootLoaderError(Exception):
     pass
 
 
-class Arguments(OrderedSet):
+class Arguments(OrderedSet):  # pylint: disable=duplicate-bases
```

Comment:
This seems to me like a pylint error. Or do you know why it is happening?

Reply:
Yes, I think it's a pylint error. `Arguments` inherits from a standard library class, and as far as I can reach, there is no sign of multiple parents.

Do you think a FIXME would be good here?

Reply:
Btw. OrderedSet is not a standard library class :-)

Reply:
Mea culpa...

Reply:
Added the FIXME for this one.